addons/skin.estuary/xml/Includes_MediaMenu.xml

```diff
@@ -273,6 +273,7 @@
 			<animation effect="fade" start="0" end="100" time="400">WindowOpen</animation>
 			<animation effect="fade" start="100" end="0" time="300">WindowClose</animation>
 			<animation effect="fade" time="300">VisibleChange</animation>
+			<animation effect="slide" end="-10,0" time="0" condition ="Window.IsActive(MyPlaylist.xml)">Conditional</animation>
```

Comment:
a param for < left > would be nicer since we wouldnt have to re-evaluate this condition during runtime then.